ping/tests/test_ping.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import mock
+
+from datadog_checks.checks import AgentCheck
+from datadog_checks.ping import PingCheck
+from datadog_checks.errors import CheckException
+
+def mock_exec_ping():
+    return """FAKEPING 127.0.0.1 (127.0.0.1): 56 data bytes
+64 bytes from 127.0.0.1: icmp_seq=0 ttl=64 time=0.093 ms
+
+--- 127.0.0.1 ping statistics ---
+1 packets transmitted, 1 packets received, 0.0% packet loss
+round-trip min/avg/max/stddev = 0.093/0.093/0.093/0.000 ms"""
+
+@mock.patch("datadog_checks.ping.PingCheck._exec_ping", return_value=mock_exec_ping())
+def test_check(aggregator, instance):
```

Comment:
When you mock a function using the `@mock.patch` decorator, the resulting mock object is automagically passed as an arg to the decorated function, in this case `mock` thinks that `aggregator` is the mocked object. You can either add an argument to the function, with the caveat that also `pytest` uses function args in a magic way so you have to check what's the resulting order once mocked, or use `mock.patch` as a context manager (see next comment below) which seems easier here.